build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml

```diff
@@ -78,10 +78,11 @@ jobs:
       PathtoPublish: '${{ parameters.nupkgdir }}'
       artifactName: 'drop'
 
-  - ${{if parameters.publishVstsFeed }}:
-    - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
-      displayName: 'NuGet push to ${{ parameters.publishVstsFeed }}'
-      inputs:
-        command: push
-        publishVstsFeed: ${{ parameters.publishVstsFeed }}
-        packagesToPush: $(Build.ArtifactStagingDirectory)/*.nupkg
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    condition:
+      ne(variables['NuGetFeed'], '')
```

Comment:
>NuGetFeed [](start = 20, length = 9)

Why switching to this variable? Can we check the parameters in the condition instead?

Reply:
`${{if parameters.publishVstsFeed}}` didn't work as expected. See this:
https://dev.azure.com/ms/microsoft-ui-xaml/_build/results?buildId=5039

I tried `${{if ne (parameters.publishVstsFeed, '')}}`, that didn't work either. I'm guessing it's some expansion problem. When NuGetFeed is not set, the variable doesn't get expanded and the value of parameters.publishVstsFeed becomes raw string `$(NuGetFeed)` so the check fails. I didn't find a good way to get around it : \

Reply:
(parameters.publishVstsFeed) I think should work? 

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#step-templates-with-parameters

Reply:
Added a check in MUX-CI.yml when passing in parameter